src/main/java/gov/cms/mat/cql_elm_translation/service/HumanReadableService.java

```diff
@@ -98,57 +136,301 @@ HumanReadableMeasureInformationModel buildMeasureInfo(Measure measure) {
+                HumanReadablePopulationModel.builder()
+                    .name("Stratification")
+                    .id(stratification.getId())
+                    .display("Stratification")
+                    .logic(getCQLDefinitionLogic(stratification.getCqlDefinition(), allDefinitions))
+                    .expressionName(stratification.getCqlDefinition())
+                    .inGroup(StringUtils.isBlank(stratification.getCqlDefinition()) ? false : true)
                     .build())
         .collect(Collectors.toList());
   }
 
-  List<HumanReadableExpressionModel> buildDefinitions(Measure measure) {
-    return List.of(new HumanReadableExpressionModel());
+  List<HumanReadableExpressionModel> buildDefinitions(Set<CQLDefinition> allDefinitions) {
+    List<CQLDefinition> definitions =
+        allDefinitions.stream()
+            .filter(definition -> definition.getParentLibrary() == null)
+            .collect(Collectors.toList());
+
+    List<HumanReadableExpressionModel> expressions =
+        definitions.stream()
+            .map(
+                definition ->
+                    HumanReadableExpressionModel.builder()
+                        .id(definition.getId())
+                        .name(definition.getName())
+                        .logic(
+                            definition
+                                .getLogic()
+                                .substring(definition.getLogic().indexOf('\n') + 1))
+                        .build())
+            .collect(Collectors.toList());
+    expressions.sort(Comparator.comparing(HumanReadableExpressionModel::getName));
+    return expressions;
+  }
+
+  List<HumanReadableExpressionModel> buildFunctions(Measure measure, String accessToken) {
+    Set<CQLDefinition> allDefinitions =
+        cqlParsingService.getAllDefinitions(measure.getCql(), accessToken);
+    List<CQLDefinition> functions =
+        allDefinitions.stream()
+            .filter(
+                definition ->
+                    definition.isFunction()
+                        && findUsedFunction(measure, accessToken, definition.getId()))
+            .collect(Collectors.toList());
+
+    List<HumanReadableExpressionModel> expressions =
+        functions.stream()
+            .map(
+                definition ->
+                    HumanReadableExpressionModel.builder()
+                        .id(definition.getId())
+                        .name(definition.getLibraryDisplayName() + "." + definition.getName())
+                        .logic(
+                            definition
+                                .getLogic()
+                                .substring(definition.getLogic().indexOf('\n') + 1))
+                        .build())
+            .collect(Collectors.toList());
+    expressions.sort(Comparator.comparing(HumanReadableExpressionModel::getName));
+    return expressions;
+  }
+
+  boolean findUsedFunction(Measure measure, String accessToken, String id) {
```

Comment:
Recommend renaming with `is` prefix. `Find` methods are commonly expected to return an Object and 'is' methods return boolean.
```suggestion
  boolean isUsedFunction(Measure measure, String accessToken, String id) {
```

Reply:
Changed